This patch should not be needed in czmq releases after 3.0.2 (merged to upstream)

--- czmq-3.0.2/src/zsys.c.orig	2015-05-26 16:25:43.000000000 +0200
+++ czmq-3.0.2/src/zsys.c	2016-03-30 02:35:49.821499607 +0200
@@ -1072,7 +1072,7 @@
         }
         //   We record the current process id in the lock file
         char pid_buffer [10];
-        snprintf (pid_buffer, sizeof (pid_buffer), "%6d\n", getpid ());
+        snprintf (pid_buffer, sizeof (pid_buffer), "%6" PRIi64 "\n", (int64_t)getpid ());
         if (write (handle, pid_buffer, strlen (pid_buffer)) != strlen (pid_buffer)) {
             zsys_error ("cannot write to lockfile: %s", strerror (errno));
             close (handle);
